scripts/check_configs.ts

```diff
@@ -596,6 +675,16 @@ async function main() {
         outcomeMap[network] = false;
       }
 
+      // query all bridges
+      const bridgesMap: BridgeMap | undefined = sdk?.token?.bridges
+      let bridgesArr: string[] = []
+
+      if (bridgesMap && bridgesMap.ibc.length && bridgesMap.polynetwork.length) {
+        const polynetworkBridges = bridgesMap.polynetwork.map((bridge) => bridge.bridgeAddresses).flat()
```

Comment:
Pls remember to check that `!!bridge.enabled` before mapping (maybe can use a `.reduce(...)` instead)